.github/workflows/build.yml

```diff
@@ -191,6 +191,7 @@ jobs:
     - name: Install cht-conf
       run:  npm install -g cht-conf
     - run: npm ci
+    - run: cd config/default && npm ci
```

Comment:
Why did this become necessary? I guess a lot of our tests use this default config so it is more surprising to me that we never had to do this before.....